flow-tests/test-express-build/test-dev-bundle-frontend-add-on/frontend/unknownfile.js

```diff
@@ -0,0 +1 @@
+// this file is not hashed in the bundle stats.json
```

Comment:
nit: newline missing
```suggestion
// this file is not hashed in the bundle stats.json

```